hilfe/handbuch/einstellungen/sonderleistungen/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layouts/hilfe/single.njk
+title: Sonderleistungen
+icon: hand-holding-box  
+---
+
+Sonderleistungen sind eine Möglichkeit einer Gruppe an Mitarbeitern gesondert vom normalen Lohn zusätzliche Zahlungen zukommen zu lassen. Unter **Einstellungen>Sonderleistungen** können Sie Kategorien anlegen um die lohn - und steuerrechtlichen Eigenschaften der Zusatzzahlungen zu definieren. 
```

Comment:
Nachdem wir jetzt ja auch für die Untermenüs icons haben, wäre es finde ich cool, wenn wir die auch in den Hilfeartikeln bei verweisen auf die entsprechenden Bereiche verwenden.

```suggestion
Sonderleistungen sind eine Möglichkeit einer Gruppe an Mitarbeitern gesondert vom normalen Lohn zusätzliche Zahlungen zukommen zu lassen. Unter {% button label="Einstellungen" icon="cog" %} / {% button label="Sonderleistungen" icon="hand-holding-box" %} können Sie Kategorien anlegen um die lohn - und steuerrechtlichen Eigenschaften der Zusatzzahlungen zu definieren. 
```